starfish/image/_stack.py

```diff
@@ -960,3 +961,91 @@ def from_numpy_array(cls, array: np.ndarray) -> "ImageStack":
                     empty.set_slice({Indices.HYB: h, Indices.CH: c, Indices.Z: z}, view)
 
         return empty
+
+    def to_pixel_intensities(self, crop: Tuple[int, int, int]=(0, 0, 0)) -> IntensityTable:
```

Comment:
Is this the right place for this method?  I kind of think this belongs more in IntensityTable than it does in ImageStack.

Reply:
Yea, that's where I originally had it. However, when I made the reversion method, that caused circular imports. So I put it here. 

Given that we're going to transfer the reversion method to the testing suite, we should be able to change this to `IntensityTable.from_image_Stack` 👍